tests/test_n_processes.py

```diff
@@ -0,0 +1,24 @@
+import fv3config
+import pytest
```

Comment:
Why did you use pytest for this test, and do we want to move other tests to pytest?

Reply:
For the tests I needed to write, pytest let me parameterize it in a very concise way, so I went for that. pytest can run unittest style tests, so it's not necessary to change the existing tests in any way.